opentreemap/importer/templates/importer/partials/merge_species.html

```diff
@@ -21,7 +21,7 @@
                             <input type="radio"
                                    id="{{ value.id }}"
                                    name="{{ field.id }}"
-                                   value="{{ value.value }}"
+                                   value="{{ value.value|unlocalize }}"
```

Comment:
Good catch! from looking at the view function I thought it was the species id, but clearly I was wrong. I'll remove this.